feat(redis): add DB menu MTA-2004 (#696)
scaleway/docs-content: databases/databases-for-redis/quickstart.mdx

---
meta:
  title: Databases for Redis™ - Quickstart
  description: This page shows you how to get started with Databases for Redis™.
content:
  h1: Quickstart
  paragraph: This page shows you how to get started with Databases for Redis™.
dates:
  validation: 2022-06-27
  posted: 2022-06-27
---

> Database for Redis™ is a low-latency caching solution based on in-memory data storage. It allows you to easily set up a secure cache and lighten the load on your main database. 

Reviewer: ```suggestion
Databases for Redis™ is a low-latency caching solution based on in-memory data storage. It allows you to easily set up a secure cache and lighten the load on your main database. 
```

> ## How to create a Database for Redis™

Reviewer: ```suggestion
## How to create a Redis™ Database Instance
```

<Message type="requirement">

  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)

</Message>

> 1. Click **for Redis™** under **Databases** on the side menu. If you do not have a database already created, the creation page displays.

Reviewer: ```suggestion
1. Click **Databases for Redis™** under **Databases** on the side menu. If you do not have a database already created, the creation page displays.
```

> 2. Click **Create Database for Redis™**. The creation wizard displays.

Reviewer: ```suggestion
2. Click **Create a Redis™ Database Instance**. The creation wizard displays.
```

    <Message type="important">

    - It is not possible to use a broadcast address (last address of the subnet, e.g. `192.168.0.255/24`) or network address (first address of the subnet, e.g. `192.168.0.0/24`).
    - It is also not possible to use the same IP twice in a Private Network.

    </Message>

    </Message>

    - **Public Network**: resources in a public network are publicly visible by default. The default Access Control List (ACL)
    settings allow anybody to connect to resources in your network. You can restrict access by configuring ACLs.

5. Complete the remaining steps in the creation wizard:
    - Create credentials: add a name and set a password for your user.
    - Enter a name and optional tags for the Database.

6. Click **Create Database for Redis™** to confirm your choices and launch creation.

## How to delete a Database for Redis™

<Message type="requirement">

- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
- You have a [Database for Redis™](/databases/databases-for-redis/how-to/create-a-database-for-redis)

</Message>

> 1. Click **Databases Redis™** under **Database** on the side menu. A list of your Database Instances displays.

Reviewer: ```suggestion
1. Click **Databases for Redis™** under **Databases** on the side menu. A list of your Database Instances displays.
```

2. Click the database name or <Icon name="more" /> > **More info** to access the database information page.

> 3. Scroll down the page to the **Delete Instance** section and click **Delete Instance**. A pop-up appears to inform that:

Reviewer: ```suggestion
3. Scroll down the page to the **Delete Database Instance** section and click **Delete Database Instance**. A pop-up appears to inform that:
```

  <Message type="important">

  This will permanently destroy your Instance and all your data will be lost. This action is irreversible.

  </Message>

> 4. Type **DELETE** to confirm and click **Delete this Instance**.

Reviewer: ```suggestion
4. Type **DELETE** to confirm and click **Delete Database Instance**.
```

2. Click **Create Database for Redis™**. The creation wizard displays.

3. Complete the following steps in the wizard:

      - Choose an Availability Zone. This is the geographical location where your database will be deployed.
      - Select a database configuration. You can choose between:
          - **Cluster mode**: creates the Database as a Cluster composed of three or more nodes. The cluster uses partitioning to split the keyspace. Each partition is replicated and can be reassigned or elected as the primary when necessary.
          - **Standalone**: creates a standalone database provisioned on a single node.
      - Select a node type.
4. Configure your Network. You can select either:
    - **Private Network**: allows your Databases to communicate in an isolated and secure network without the need for a public IP address. If you select this option you must: 1. select a Private Network to attach from the drop-down and 2. enter an IP address for each node in your Private Network. The IP addresses must belong to the same subnet. The mask you select applies to all IP addresses.

    <Message type="important">

>     The Private Network feature with Database for Redis™ is only available with the [cluster mode](/databases/databases-for-redis/concepts). If you want to use Private Networks you must:

Reviewer 2: ```suggestion
    If you want to use Private Networks you must:
```
